api/resources/permissions.py

```diff
@@ -41,6 +41,14 @@ def has_permission(self, request, view):
         return proxy_object.has_permission(auth.user, self.REQUIRED_PERMISSIONS[request.method])
 
 
+class ResourceListPermission(ResourcesPermission, permissions.BasePermission):
+    '''Permissions for the top-level ResourceList endpoint.
+
+    ResourceList only supports POST
+    '''
+    REQUIRED_PERMISSIONS = {'POST': 'admin'}
```

Comment:
Can write contributors not add resources?

Reply:
Let's clarify in Scrum today!